docs/getting_started/basics/execute.md

```diff
@@ -4,32 +4,52 @@ The execution of a node in a Shift workflow consist in the evaluation of its int
 
 There are different ways a workflow can be executed, which depend on which nodes will get computed. 
 
-## ![Execute All Button](../../images/toolbar/execute_all_default.svg){:width="25" height="25"} Execute All
+## ![Execute All Button](images/execute_all_default.svg){style="width:5%"} Execute All
 
 The "Execute All" action will compute all the dirty nodes present in the workflow. 
 
-![Execute All](../../images/execute_all.gif)
+<figure>
+      <img src="images/execute_all.gif" alt="Execute All">
```

Comment:
In this gif still appears the old "Operator Name" naming in the inspector, should we update it?

Reply:
Changed 
https://github.com/Inbibo/shift_docs/pull/61/commits/69e91498ee0d9a270c8014ac34740c78c4906515